src/main/webapp/script.js

```diff
@@ -211,3 +214,35 @@ for (let button of filterButtons) {
       }
     });
 }
+
+let infowindow;
+function updateRestaurantInfo(restaurant, marker) {
+  if (!infowindow) {
+    infowindow = new google.maps.InfoWindow();
+    infowindow.setZIndex(500); // Random high number so it shows on top
+  }
+  let { geometry: {location}, placeId, name, photos, rating } = restaurant;
+
+  let params = {
+    api: 1,
+    destination: name,
+    destination_place_id: placeId
+  }
+  const dir_url = new URL('https://www.google.com/maps/dir/');
+  Object.keys(params).forEach(key => dir_url.searchParams.append(key, params[key]));
+
+  let contentString =
+    `<div id="info-container">
+      <div id="info-name">${name}</div>
+      <div id="info-body">
+        <a target="_blank" href="${dir_url}">
+          <img src="${directionIconUrl}">
+        </a>
+      </div>
+    </div>`;
```

Comment:
Inconsistenct with how the other infowindow was implemented. Any idea which one is better?

Reply:
No idea, but I will have it implemented the way it was with the first infowindow for consistency, thanks!